person.rb

```diff
@@ -19,3 +29,10 @@ def can_use_services?
 
```

Comment:
![image](https://github.com/Lawmsangi/school-library-oop/assets/93039359/835f6a31-5f5a-4bb3-9a03-5a066197ce11)

- [ ] It would be nice if you fix the highlighted link in the image above ⬆️ because it is broken 😨 Kindly fix it or remove it if you don't have any dedicated section for this link 🤔